git_services/git_services/init/cloner.py

```diff
@@ -68,6 +68,11 @@ def _make_dirname(url: str) -> str:
 
 
 class GitCloner:
+    mount_path: Path
```

Comment:
issue: This is not a dataclass but we are kinda using it like one. The stuff that is already there was hardcoded constant type stuff as class variables. But I think it is confusing to have the rest as class variables - and it may lead to bugs. So lets make it a dataclass.